factcast-core/src/main/java/org/factcast/core/spec/FactSpec.java

```diff
@@ -48,6 +52,22 @@ public class FactSpec {
 
   @JsonProperty FilterScript filterScript = null;
 
+  public Set<UUID> aggIds() {
```

Comment:
unexpected behavior when used to modify from the outside.
We should wrap into unmodifiable